### ISSUE TITLE
batch_size

### ISSUE THREAD

**Issue by LSiyN** [Camaltra/road_segementation#1]
What size of learning should be set for a 1024 * 1024 dataset_ Rate batch_ Size and num_ Epoch

**Comment by Camaltra** [comment]
Hello, thank for your question 🙏 

For an image patch of 1024 x 1024, the batch_size will depend of your amount of available RAM on your computer. For exemple I run on a 256 x 256 patches image size and use a batch size of 16. On the metrics of my computer it fitted well, and may have enough RAM to get with a higher batch_size.

For the num of epochs, I choose 100 firstly and let the learning run for 3/4 hours and checked the results. They were satisfy
for me, and so didn't go further on the training, but if you want to get a better Dice Score, I think you can let it run a bit longer, it will surly improve the model.

For the learning rate, I picked 1e-3 which fit well with a small batch size, but if you get with a lower batch size (As if you get with a 1024 x 1024 images patches size), it's preferable to lower a bit the learning rate to not overfit on a single batch, and get with 1e-4 or even use other techniques as dropout (Need to be added directly on the model)

If you have any other question feel free to respond here :)

**Comment by LSiyN** [comment]
Why does the loss function not converge?

**Comment by Camaltra** [comment]
What are the parameters you used ?

I could be due to different one, and may need to figure out what are responsible for, and try to change them
